domain/status/status.go

```diff
@@ -107,6 +108,8 @@ func EncodeRelationStatus(s RelationStatusType) (int, error) {
 		return 3, nil
 	case RelationStatusTypeSuspended:
 		return 4, nil
+	case RelationStatusTypeError:
+		return 4, nil
```

Comment:
todo: make status values unique: s/4/5/.

Reply:
Oops, that is a pretty bad typo, good catch.